backend/app/services/integration_service.py

```
@@ -143,7 +142,7 @@ async def update_integration(
 
             existing = await self.get_integration(user_id, integration_id)
             if not existing:
-                raise NotFoundError("Integration not found")
+                raise ValueError("Integration not found")
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)